drivers/sensor/scd30/Kconfig

```diff
@@ -17,12 +21,6 @@ config SCD30_MEASUREMENT_INTERVAL
       The interval with wich measurement are done.
       Has to be in the range [2; 1800].
 
-config SCD30_ENABLE_ALTITUDE_COMPENSATION
-    bool "Enable altitude compensation"
-
-config SCD30_ENABLE_OFFSETS
-    bool "Enable temperature and RH offsets"
-
```

Comment:
you just added those in previous commit, and now you remove them again.

Please cleanup your commits by squashing the commits.